daml-lf/validation/src/main/scala/com/digitalasset/daml/lf/validation/Typing.scala

```diff
@@ -399,6 +416,12 @@ private[validation] object Typing {
       }
     }
 
+    def checkDefException(excepName: TypeConName, defException: DefException): Unit = {
+      val DefException(message) = defException
```

Comment:
Shouldn't we also check here that the type is a record type without any type parameters?

Reply:
I checked it in the pastern matching line 283, just before calling `checkDefException`.

Reply:
I see. Thanks for the explanation.

Reply:
both conditions are tested in `daml-lf/validation/src/test/scala/com/digitalasset/daml/lf/validation/TypingSpec.scala`